.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 120
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 10
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - kind/feature
+  - kind/enhancement
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
This doesn't seem correct, as the bot may close fixable bug issues.
Maybe `staleLabel: stale` or `staleLabel: closed-due-to-inactivity`?

Reply:
indeed, copy/pasted the reference doc, but should better adopt `stale` we use [here](https://github.com/docker/compose/blob/v2/.github/stale.yml#L27)